sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## 7.0.0 (Unreleased)
 
+- Log message for the `actionAfterWaitTimeout`(gets triggered for receiveMessages API) has been updated to reflect the proper cause of batch receive termination.
```

Comment:
Logging is internal details and we dont need to capture every change in a log statement in the changelog

Reply:
It felt like a bug fix, hence added changelog.
Removed now.